README.md

```diff
@@ -140,6 +140,11 @@ You can deploy this project using:
 - Twitter: [@amiralda](https://twitter.com/amiralda)
 - LinkedIn: [@dany-augustin](https://linkedin.com/in/dany-augustin)
 
+👤 **Daniel Villalba Lynch**
+
+- GitHub: [@gdvl777](https://github.com/gdvl777)
+- LinkedIn: [Daniel Villalba Lynch](https://linkedin.com/in/dvillalba777)
+
```

Comment:
Great design team 🤝

- [ ] [Optional] Kindly remove this author from your readme file since he didn't contribute to this pull request.
![danny](https://github.com/AndreaM2429/Portfolio_setup_and_mobile_first/assets/63917725/bf18c8d3-16df-41e2-8838-608ab2942b33)
- [ ] [Optional] Kindly ensure you are referring to this project instead of the `Hello Microverse` project.
![setup](https://github.com/AndreaM2429/Portfolio_setup_and_mobile_first/assets/63917725/ec47e618-a597-4129-abe3-92a2467fe4b5)